ellie/monopoly.py

```diff
@@ -0,0 +1,154 @@
+from random import randint
+
+class Player:
+    def __init__(self, player_name, player_token):
+        self.name = player_name
+        self.token = player_token
+        self.money = 1500
+        self.properties = []
+        self.position = 0
+        self.in_jail = False
+        self.is_turn = False
+    
+    def __repr__(self):
+        return f"This player, {self.name}, is playing as the {self.token}. \n{self.name} has ${self.money}, is currently on {board[self.position]}, and owns these properties: {self.properties}"
+
+    def roll_dice(self):
+        # Roll two dice and return the results
+        die1 = randint(1,6)
+        die2 = randint(1,6)
+        result = die1 + die2
+        print(f"{self.name} rolled a {result} with a {die1} and a {die2}!")
+        return [die1, die2, result]
```

Comment:
Fixed-length groups in Python usually use [tuples](https://docs.python.org/3/library/stdtypes.html#tuple) since they cannot be modified.